src/tools/sfdk/command.cpp

```diff
@@ -72,6 +72,7 @@ const char VM_MEMORY_SIZE_MB[] = "vm.memory-size";
 const char VM_SWAP_SIZE_MB[] = "vm.swap-size";
 const char VM_CPU_COUNT[] = "vm.cpu-count";
 const char VM_STORAGE_SIZE_MB[] = "vm.storage-size";
+const char VM_FREE_STORAGE_SIZE_MB[] = "vm.free-storage-size";
```

Comment:
> Previously we discussed that this wouldn't be documented, at least for the time being. This was supposed to be used by the installer only.

Right, and that is great because only now I realize this would not be easy to document consistently with the existing properties. So yes, please omit the docs change.

```suggestion
const char VM_FREE_STORAGE_SIZE_MB[] = "vm.free-storage-size"; // intentionally undocumented, write-only
```